src/components/dataPanel/PanelViewingState.tsx

```diff
@@ -56,16 +55,16 @@ const OneColumn: FunctionComponent<IPanelViewStateProps> = ({ fields }) => (
             field.value instanceof Array ? field.value : [field.value];
           return (
             <TableRow key={index}>
-              <TableCell component="th" scope="row">
+              <TableCellWithoutLines component="th" scope="row">
```

Comment:
Remove lines to match wireframe.  

Also makes visual change less jarring when in editing state, as the Material-UI input components exceed the standard table height. Notice that the three fields on the left (editing) do not align vertically with the fields on the right (viewing).  I chose not to go for pixel perfection here...

![image](https://user-images.githubusercontent.com/54271433/117648409-14f0ee00-b186-11eb-919c-8066da71616e.png)